Peoplesoft/variables.tf

```diff
@@ -58,17 +71,21 @@ variable "compute_hostname_prefix_es" {
   default = "es"
 }
 
-
 variable "compute_hostname_prefix_ps" {
-  description = "Elastic Search server host resource prefix"
+  description = "Process server host resource prefix"
   default = "ps"
 }
 
 variable "compute_hostname_prefix_tc" {
-  description = "Elastic Search server host resource prefix"
+  description = "Tools Client server host resource prefix"
   default = "tc"
 }
 
+variable "compute_hostname_prefix_id" {
+  description = "Identity server host resource prefix"
+  default = "id"
+}
+
 # Set instance counts. 
 #Min should be two for Application, Webserver, Elastic and ProSched due to Availablity Set usage.
 variable "compute_instance_count" {
```

Comment:
Let's ask for only 1 count and remove the rest similar to what we're doing with the VM size